cmd/parca-agent/main.go

```diff
@@ -75,9 +72,10 @@ type flags struct {
 	SamplingRatio      float64           `kong:"help='Sampling ratio to control how many of the discovered targets to profile. Defaults to 1.0, which is all.',default='1.0'"`
 	Kubernetes         bool              `kong:"help='Discover containers running on this node to profile automatically.',default='true'"`
 	PodLabelSelector   string            `kong:"help='Label selector to control which Kubernetes Pods to select.'"`
-	Cgroups            []string          `kong:"help='Cgroups to profile on this node.'"`
-	// SystemdUnits is deprecated and will be eventually removed, please use the Cgroups flag instead.
-	SystemdUnits []string `kong:"help='[deprecated, use --cgroups instead] systemd units to profile on this node.'"`
+	// Cgroups is deprecated and will be eventually removed.
+	Cgroups []string `kong:"help='[deprecated, will automatically profile all cgroups] Cgroups to profile on this node.'"`
```

Comment:
we're pre v1 and this is already unused now, so I would recommend to just remove it instead

Reply:
Happy to do that! Thought that it could be confusing for users that might not be aware of these changes to see that not only their chosen cgroup is profiled